quickwit-storage/src/storage.rs

```diff
@@ -121,4 +121,7 @@ pub trait Storage: Send + Sync + 'static {
 
     /// Returns an URI identifying the storage
     fn uri(&self) -> String;
+
+    /// Return the root path of this storage if appropriate.
```

Comment:
I really wonder why you would need that. the comment is not helping... is it to guess where the cache should be created?

Reply:
Yes, mostly for SingleFileStorage, I will elaborate.